sw/battor.c

```diff
@@ -193,6 +196,7 @@ int main(int argc, char** argv)
 		sconf.ovs_bits = ovs_bits;
 		// TODO set proper gain!
 		sconf.gain = eeparams->gainL;
+		param_get_rtc_for_file(down_file);
```

Comment:
TODO: this should return the RTC, which will then be stored in sconf and added to each sample when printed